modules/learn/pages/security/certificates.adoc

```diff
@@ -263,15 +266,39 @@ For step-by-step instructions, see xref:manage:manage-security/enable-client-cer
 == Deprecation of Subject Common Name
 
 Use of the _Subject Common Name_ to identify either a server or a client is now deprecated (see https://tools.ietf.org/html/rfc6125#section-6.4.4[section 6.4.4 of RFC 6125^]).
-For Couchbase Server, this means that:
+For Couchbase Enterprise Server Version 7.2 and later, this means that:
 
-* The node-certificate for each server in the cluster is recommended to specify at least one _Subject Alternative Name_, in order to be identified: typically, the node's IP address or DNS name should be provided in this way.
+* The node-certificate for each server in the cluster must specify its node-name as a _Subject Alternative Name_: see xref:learn:security/certificates.adoc#server-certificate-validation[Server Certificate Validation], immediately below, for details.
 Subject Common Name may continue to be specified.
 
 * A client-certificate may continue to specify only a Subject Common Name.
 
 Examples of certificate-creation provided in xref:manage:manage-security/manage-certificates.adoc[Manage Certificates] continue to include definitions of Subject Common Name for both server and client.
 
+[#node-certificate-validation]
+== Node Certificate Validation
+
+In Couchbase Enterprise Server Version 7.2 and later, each _node certificate_ must be configured with the node-name correctly specified as a _Subject Alternative Name_ (SAN).
+A check is performed when the certificate is loaded onto the node; and, after the certificate has been loaded, a check will also be performed whenever an attempt is made to add or join the node to a cluster.
+If the node-name is either incorrectly specified as a SAN, or is missing:
+
+* If the name of the node itself can be changed, a warning is generated, and the operation succeeds.
+
+* If the name of the node itself cannot be changed, an error occurs, and the operation fails.
+
+See xref:learn:clusters-and-availability/nodes.adoc#naming-clusters-and-nodes[Naming Clusters and Nodes], for more information on naming conventions and restrictions.
+
+The following options are available, for specifying the node-name as a SAN.
```

Comment:
It is a bit incorrect I think.
If node-name is FQDN, then "DNS:..." SAN must be present (IP is not an option). Wildcards are ok though.
If node-name is IP, then "IP address:..." SAN must be present (DNS is not an option).

Reply:
Okay, I've had a go at modifying this, specifying the restrictions more clearly.